crates/ruff/resources/test/fixtures/ruff/RUF017.py

```diff
@@ -0,0 +1,12 @@
+x = [1, 2, 3]
+y = [4, 5, 6]
+
+# RUF017
+sum([x, y], start=[])
+sum([x, y], [])
+sum([[1, 2, 3], [4, 5, 6]], start=[])
+sum([[1, 2, 3], [4, 5, 6]], [])
```

Comment:
can you a case where the expression is multiline? your code should work but the case should be in the tests

Reply:
Added

Reply:
Is it intentional that you didn't push the change yet?

Reply:
Should be right below this line